main.py

```diff
@@ -5087,8 +5087,8 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
                             cur.execute("INSERT INTO godimage_requests (requesterid, waifuid, image, is_global, state, created) VALUES(%s, %s, %s, %s, 'pending', %s)", insertArgs)
 
                             # notify the discordhook of the new request
-                            waifuInfo = getWaifuById(waifuid)
-                            discordArgs = {"user": tags['display-name'], "id": waifuid, "name": waifuInfo["name"], "image": args[2], "type": "a global" if do_global else "an"}
+                            waifu = getWaifuById(waifuid)
+                            discordArgs = {"user": tags['display-name'], "id": waifuid, "name": waifu["name"], "image": args[2], "type": "a global" if do_global else "an"}
```

Comment:
Wording: Maybe it should be "an individual"/"a global" rather than just implicitly stating "an image change" for an individual image change. The more explicit wording might prevent misunderstandings, especially if hastily reading the message on the go.